core/src/test/java/org/elasticsearch/index/query/BaseQueryTestCase.java

```diff
@@ -198,17 +200,17 @@ public void afterTest() {
      */
     @Test
     public void testFromXContent() throws IOException {
-        QB testQuery = createTestQueryBuilder();
+        QB expectedQuery = createTestQueryBuilder();
```

Comment:
not sure about this name change to be honest. This query is the one that we use for testing, we start with it, we convert it to string, we reparse it and we compare the result with newQuery, which is actually the expected query.